Documentation/ApiOverview/DatabaseRecords/Index.rst

```diff
@@ -0,0 +1,110 @@
+for managing content and data within the system.
+
+..  contents::
+    :caption: Content on this page
+
+.. _database-records-examples:
+
+Common examples of records in TYPO3:
+====================================
+
+Page records
+    These represent pages in the page tree, which structure the website. They
+    are stored in table :sql:`pages`.
+Content records
+    These include text, images, videos, or any other media placed within a
+    page. They are stored in table :sql:`tt_content`.
+Backend user records
+    Information about users who have access to the TYPO3 backend. They
+    are stored in table :sql:`be_users`. The backend user groups they are
+    organized in are stored in table :sql:`be_groups`.
+System records
+    System records control the configuration and management of the TYPO3 system.
+    Examples include file references, file mounts, or categories.
+Extension-specific records
+    Extensions often define custom records to store specific data, such as
+    products for a shop system or events for a calendar.
+
+..  todo: Create a page listing all tables created by the core and explain what
+    they do. Link from here
+
+.. _database-records-technical:
+
+Technical structure of a record:
+================================
+
+Each record is stored in a database table. Each row represents one record. Each
+column represents a field of the record or some kind of metadata.
+
+A record typically includes a unique identifier in field `uid`, the id of the
+page record on which it is located in field `pid`, fields for various
+attributes (for example, title, content), and metadata like creation and
+modification timestamps, visibility, information on translation
+and workspace handling. A record can have relations to other records.
+
+.. _database-records-tca:
+
+TCA (Table Configuration Array)
+===============================
+
+TYPO3 uses the **TCA** to define how records of a specific table are
+structured, how they are displayed in the backend, and how they interact
+with other parts of the system. See the :ref:`TCA Reference <t3tca:start>`
+for details.
+
+.. _database-records-types:
+
+Types and subtypes in records
+-----------------------------
+
+In TYPO3, different types and subtypes of records are often stored in the same
+database table, even though not all types share the same columns. This approach
+allows for flexibility and efficiency in handling diverse content and data
+structures within a unified system.
+
+TYPO3 uses a **single-table inheritance** strategy, where records of various
+types are distinguished by a specific field, often named :sql:`type`.
+For historical reasons the field is named :sql:`CType` for content elements
+and :sql:`doktype` for pages. The field that is used for the type is defined in
+TCA in :confval:`ctrl > type <t3tca:ctrl-type>`. The types itself are stored in
+:ref:`types <t3tca:types>`.
+
+This allows TYPO3 to store related records, such as pages or content elements,
```

Comment:
I don't understand this sentence. Sounds like pages and content elements can be stored in one shared table.

Reply:
Changed it